### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by mfadhili**
Project is complete with linters and the styles done

### CODE REVIEW

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# INVALID ❌ ❌ ❌ 

Hi @mfadhili, 👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌
 - Your linters tests are failing. Please make sure all the tests are passing before submitting a PR for review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

----
_Invalid Code Review Request does not count into the code reviews limit._

**Comment by mfadhili** [comment]
correcting index file with width content

File: styles.css [hunk 1]
```diff
@@ -0,0 +1 @@
+body { background-color: azure; }
```
Comment (Abenezer-Tilahun) [inline styles.css]: - [x] As provided in the project requirements, kindly give your h1 element in your index.html have a class and use a class selector to style it.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node_modules
+hint_report
```
Comment (Abenezer-Tilahun) [inline .gitignore]: - [x] As provided in the project requirements, kindly Add a "test.md" file with the text "This file should be ignored by git". please Make sure that the last file("test.md" file) is actually ignored by git and is not present in your GitHub repository.

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @mfadhili    , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Good README
✔️ Good commit messages

## Required Changes ♻️
_Check the comments under the Review.
## Optional suggestions

- [x] Kindly Add descriptive README file to your project but remember to customize it to your project. please add your GitHub and LinkedIn username at the author section and fix the [MIT license](https://choosealicense.com/licenses/mit/) in your Readme file, The Link is broken.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mfadhili** [comment]
- Added the class CSS selector to the header
- included `test.md` in the project and included it in `.gitignore`

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @mfadhili , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :heavy_check_mark: 
:heavy_check_mark: No linter errors
:heavy_check_mark: Good commit messages

## Optional suggestions
[Optional ] 
- As mentioned by the previous reviewer kindly consider customizing your readme file to your project and fixing the link to the `MIT` license in your Readme file, the current link is broken. You can fix this by applying the following line of markdown on the license link.
``` 
[MIT](./MIT.md)
 ```
Current screenshot when I click on the link

Screenshot from 2022-09-22 20-24-57

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mfadhili** [comment]
Thank for the reviews @Abenezer-Tilahun , @leonardodiasb , @Felix45

**Pull request merged**